### PR TITLE
What's New updates Sept 15 - Sept 30

### PR DESCRIPTION
**Pull request by bdenham**
## Purpose of this pull request

This pull request (PR) updates the what's new page by incorporating additions from Sept 15 - Sept 30 (today)

## Screenshot of Build

image

### CODE REVIEW

File: src/_data/whats-new.yml [hunk 1]
```diff
@@ -1,11 +1,63 @@
...
+updated: Wed Sep 30 08:45:34 2020
 entries:
+- description: Added the Import section to the [Contribution Guide](https://devdocs.magento.com/contributor-guide/contributing.html#import).
+  versions: ''
+  type: Major Update
+  date: September 24, 2020
+  link: https://github.com/magento/devdocs/pull/7886
+  contributor: sidolov
+  profile: https://github.com/sidolov
+- description: Added an optional step to show how to add default values in the [Creating
```
Comment (jeff-matthews) [inline src/_data/whats-new.yml:16]: Original PR description needs a period.

**Review by jeff-matthews — CHANGES_REQUESTED** [review]

File: src/_data/whats-new.yml (hunk 1 above)
Comment (hguthrie) [reply src/_data/whats-new.yml:16]: PR 7918 does have a period at the end. I wrote that one. Is something not appearing correctly?

Comment (jeff-matthews) [reply src/_data/whats-new.yml:16]: Interesting... it should show up in the output in this PR then. Not sure what's up with that.

Comment (jeff-matthews) [reply src/_data/whats-new.yml:16]: Oh wait, my mistake. The line breaks are throwing me off. No change needed.

**Review by jeff-matthews — APPROVED** [review]

File: src/_data/whats-new.yml (hunk 1 above)
Comment (hguthrie) [reply src/_data/whats-new.yml:16]: ahhh..okay. whew!

**Review by dshevtsov — DISMISSED** [review]
The `versions:` is empty in several entries. It should be set to `2.x` unless specified. Please ask assignees at the corresponding PRs to add labels to indicate target/affected versions.

**Comment by bdenham** [comment]
running tests

**Pull request merged**